src/main/java/tds/content/controllers/ContentController.java

```diff
@@ -45,11 +52,32 @@ public ContentController(final ItemDocumentService itemDocumentService) {
         ITSDocument itemDocument;
 
         try {
-            itemDocument = itemDocumentService.loadItemDocument(new URI(itemPath), accLookup);
+            itemDocument = contentService.loadItemDocument(new URI(itemPath), accLookup);
         } catch (URISyntaxException e) {
             throw new IllegalArgumentException(String.format("The provided item path '%s' was malformed", itemPath));
         }
 
         return ResponseEntity.ok(itemDocument);
     }
+
+    @GetMapping(value = "/resource")
+    @ResponseBody
+    public ResponseEntity<Resource> getResource(@RequestParam final String resourcePath) throws IOException {
+        InputStream inputStream;
+
+        try {
+            inputStream = contentService.loadResource(new URI(resourcePath));
+        } catch (URISyntaxException e) {
+            throw new IllegalArgumentException(String.format("The provided resource path '%s' was malformed", resourcePath));
+        }
+
+        final HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_OCTET_STREAM);
+        ByteArrayResource resource = new ByteArrayResource(IOUtils.toByteArray(inputStream));
```

Comment:
I think the InputStreamResource is the better option since the byte array will always be the entire object in memory.  The input stream could be a buffered input stream rather than forcing everything into memory.